File: swarmmanagercontroller/src/main/java/co/uk/swarmbit/docker/api/secrets/parameters/SecretsFilters.java
package co.uk.swarmbit.docker.api.secrets.parameters;

import co.uk.swarmbit.docker.api.common.AbstractFilters;
import co.uk.swarmbit.docker.api.common.Filters;

public class SecretsFilters extends AbstractFilters implements Filters {


    public SecretsFilters setId(String id) {
        addFilter("id", id);
        return this;
    }

    public SecretsFilters setName(String name) {
        addFilter("name", name);
        return this;
    }

    public SecretsFilters setLabel(String label) {
        addFilter("label", label);
        return this;
    }

}

File: swarmmanagercontroller-new/src/main/kotlin/com/swarmbit/docker/api/secrets/parameters/SecretsFilters.kt
package com.swarmbit.docker.api.secrets.parameters

import com.swarmbit.docker.api.common.AbstractFilters

class SecretsFilters : AbstractFilters() {

    fun setId(id: String): SecretsFilters {
        addFilter("id", id)
        return this
    }

    fun setName(name: String): SecretsFilters {
        addFilter("name", name)
        return this
    }

    fun setLabel(label: String): SecretsFilters {
        addFilter("label", label)
        return this
    }

}